packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -67,7 +67,7 @@ export class OdspDocumentServiceFactoryCore
 		return this.nonPersistentCache.snapshotPrefetchResultCache;
 	}
 
-	public get IRelaySessionAwareDriverFactory() {
+	public get IRelaySessionAwareDriverFactory(): this {
```

Comment:
That would be a type-wise breaking change, so I'd prefer not to do it here. I'll leave a TODO, though.
```suggestion
	// TODO: return `IRelaySessionAwareDriverFactory` instead of `this` (breaking change)
	public get IRelaySessionAwareDriverFactory(): this {
```